### PR TITLE
Anica - Cohort Manager Challenge

### PR DESCRIPTION
**Pull request by Anicaaa**

### CODE REVIEW

File: src/manager.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    const cohort = {
+      name: cohortName,
+      studentList: []
+    }
+    this.cohortList.push(cohort)
+    return this.cohortList
+  }
+
+  searchCohortName(cohortName) {
+    const newArray = []
```
Comment (d-username) [inline src/manager.js:16]: on this one, you created a newArray, and you return the newArray...but when you search for a cohort by name in this case you would only return 1 cohort - so you would not need to create a newArray - just return the found cohort by itself.

File: src/student.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class Student {
+  constructor(studentId, firstName, lastName, githubUsername, email) {
+    this.studentId = 1
```
Comment (0xtobycarlo) [inline src/student.js]: I did `this.studentID = id` here instead, made it easier for me to assign new ID's to new students that I created

File: src/student.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    githubUsername: "Anicaaa",
+    email: "anica@gmail.com",
+  },
+  {
+    studentId: 2,
+    firstName: "David",
+    lastName: "Divad",
+    githubUsername: "Daviddd",
+    email: "david@gmail.com",
+  },
```
Comment (d-username) [inline src/student.js]: heeey, that's my name here :D

File: spec/manager.spec.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+const Manager = require("../src/manager.js");
+const Cohort = require("../src/cohort.js");
+const Student = require("../src/student.js");
+
+describe("Manager", () => {
+  it("create a cohort name", () => {
```
Comment (0xtobycarlo) [inline spec/manager.spec.js:6]: Screenshot 2022-04-04 at 10 06 48
You have a test for creating a cohort name but do you have a test for creating the cohort itself? I would create a new cohort with:  
   `const expected = ['Cohort x']`
   `const result = cohortManager.createCohort('Cohort x')`
   `expect(result).toEqual(expected)`
And then create the cohort itself below

File: spec/manager.spec.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+      {
+        name: "Cohort 2",
+        studentList: [],
+      },
+    ];
+    const result = manager.removeCohortName("Cohort 1");
+    //verify
+    expect(result).toEqual(expected);
+  });
+});
```
Comment (0xtobycarlo) [inline spec/manager.spec.js:59]: looks good :)

**Pull request closed without merging**